style.css

```diff
@@ -164,6 +169,106 @@ ul {
   justify-content: space-between;
 }
 
+/* -------------POPUP WINDOW------------ */
+
```

Comment:
- [ ] I can see that you have done a great job adding popup window. However something looks broken in mobile screen view, please check the video below: 

https://user-images.githubusercontent.com/77038610/146387171-7dd5d880-da05-4280-92db-b4e88a0460d5.mp4

As you can see, width given to `<h2>` element is causing issues, you can solve it just by removing the `width: 801px;`